### DISCUSSION TITLE
How to use the Module

### DISCUSSION THREAD

**Discussion by Dadrak** [EddieDover/fvtt-player-achievements#13] [Support] [answered]
Hi there, love the idea of achievements

But I wanted to know if there is a way to award achievements to players ? Is there a manual way for me as a GM to do so ?

again thanks man, have a nice day :)

**Comment by EddieDover** [comment, marked as answer]
I'll preface this by saying that I'll admit I'm not the best when it comes to user experience. And I've stared at this project for so long that I'm pretty sure there's a lot of developer-bias.  If you think there is a cleaner way to present things, or if you'd like to modify the README and make things more clear for other users, I would welcome it. (I'm not being sarcastic at all)

When you open the achievements panel as a GM, it should look like this:
I've underlined the relevant section that should appear, as long as there are players currently logged in.  All you need to do is click their name to publicly award the achievement.
![image](https://github.com/EddieDover/fvtt-player-achievements/assets/464286/98cb4ec7-a613-4f17-a079-59b023bf63e5)
